src/Compilers/Core/Portable/Operations/OperationNodes.cs

```diff
@@ -8836,16 +8958,19 @@ public override IBlockOperation ExpressionBody
 
     internal sealed class DiscardOperation : Operation, IDiscardOperation, IPatternOperation
```

Comment:
>IDiscardOperation, IPatternOperation [](start = 56, length = 36)

This looks surprising. In master, IDiscardOperation and IPatternOperation are not related. Are they related in recursive-patterns branch? Was that an explicit design decision? IOperation nodes should not implement unrelated interfaces.

Reply:
This has already been noted and addressed in https://github.com/dotnet/roslyn/pull/32018